utils/build_swift/tests/build_swift/test_driver_arguments.py

```diff
@@ -107,7 +107,7 @@ def test(self):
             with utils.redirect_stdout() as output:
                 with self.assertRaises(ParserError):
                     self.parse_args([option.option_string])
-                    self.assertNotEmpty(output)
+                self.assertTrue(output)
```

Comment:
`assertTrue` checks whether `bool(x)` is true, I found it recommended as the way to check whether a collection is non-empty, since `if container` is a standard idiom for checking non-empty containers.